backend/director/llm/openrouter.py

```
@@ -0,0 +1,205 @@
+
+    def __init__(self, config: OpenRouterConfig = None):
+        """
+        :param config: OpenRouter Config
+        """
+        if config is None:
+            config = OpenRouterConfig()
+        super().__init__(config=config)
+        try:
+            import openai
+        except ImportError:
+            raise ImportError("Please install OpenAI python library.")
```

Ruff (0.15.9)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)